includes/utilities.inc

```diff
@@ -310,51 +309,35 @@ EOQ;
  */
 function islandora_newspaper_set_mods_date_issued(AbstractDatastream $datastream, DateTime $date) {
   $out = FALSE;
-  $file = file_create_filename("{$datastream->parent->id}_{$datastream->id}.xml", 'temporary://');
-  $datastream->getContent($file);
-  @$doc = simplexml_load_file($file);
-  if ($doc) {
-    $doc->registerXPathNamespace('ns', 'http://www.loc.gov/mods/v3');
+  $dom = new DOMDocument();
+  $dom->loadXML($datastream->content);
+  $xpath = new DomXPath($dom);
+  if ($dom) {
```

Comment:
Should always be TRUE, could be the return value from loadXML.  The change might regress the test case?